lib/charms/ovn_charm.py

```diff
@@ -1182,3 +1182,51 @@ def render_nrpe(self):
         nrpe.add_init_service_checks(
             charm_nrpe, self.nrpe_check_services, current_unit)
         charm_nrpe.write()
+
+    def configure_iptables_rules(self):
+        """
+        Configure `iptables` NOTRACK rules for GENEVE traffic in nf_conntrack
+
+        The GENEVE/VXLAN traffic has randomized source ports[1], which
+        plays badly with nf_conntrack in a busy cloud environment. Having
+        randomized source ports change the five-tuple info, resulting to an
+        increase in unique connections that nf_conntrack tracks. That
+        inevitably leads to a full nf_conntrack table and connections to be
+        dropped.
+
+        NOTRACK rules allow nf_conntrack to ignore the said traffic. UDP
+        flows with destination port 6081 will not be tracked by nf_conntrack
+        while rules are in effect.
+
+        [1]:(https://datatracker.ietf.org/doc/html/rfc8926#section-3.3)
+        """
+        ch_core.hookenv.log("Configuring iptables rules")
```

Comment:
Please be explicit about which log level is used for all calls to `log`.

Reply:
Noted, done.